scenarios/topo/pfcp-slice-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "sliceName" : "MOBILE",
+  "sliceName" : "P4-UPF-1",
```

Comment:
do we need to verify if these params are actually set ?

Reply:
Added a verification of the meter band in the step where we verify the slice meter.